docs/source/release_notes.rst

```diff
@@ -9,9 +9,11 @@ Release Notes
         * Fixes
         * Changes
         * Documentation Changes
+            * Add time series guide (:pr:`1896`)
```

Comment:
Need to uncomment the Future Release section, but that should be resolved when you pull in the latest from `main`.